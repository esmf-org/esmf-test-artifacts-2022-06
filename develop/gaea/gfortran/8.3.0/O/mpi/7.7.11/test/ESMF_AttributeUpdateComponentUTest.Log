build time -- 2022-04-09 00:42:29
20220409 060027.322 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.322 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.322 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.322 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.322 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.322 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.322 INFO             PET0 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.322 INFO             PET0 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.322 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.322 INFO             PET0 ESMF_COMM                   : mpi
20220409 060027.322 INFO             PET0 ESMF_MOAB                   : enabled
20220409 060027.322 INFO             PET0 ESMF_LAPACK                 : enabled
20220409 060027.322 INFO             PET0 ESMF_NETCDF                 : enabled
20220409 060027.322 INFO             PET0 ESMF_PNETCDF                : disabled
20220409 060027.322 INFO             PET0 ESMF_PIO                    : enabled
20220409 060027.322 INFO             PET0 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET0  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET0  PASS  PET            0 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET0  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET0 Finalizing ESMF
20220409 060027.318 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.318 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.318 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.318 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.318 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.318 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.318 INFO             PET1 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.318 INFO             PET1 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.318 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.318 INFO             PET1 ESMF_COMM                   : mpi
20220409 060027.319 INFO             PET1 ESMF_MOAB                   : enabled
20220409 060027.319 INFO             PET1 ESMF_LAPACK                 : enabled
20220409 060027.319 INFO             PET1 ESMF_NETCDF                 : enabled
20220409 060027.319 INFO             PET1 ESMF_PNETCDF                : disabled
20220409 060027.319 INFO             PET1 ESMF_PIO                    : enabled
20220409 060027.319 INFO             PET1 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET1  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET1  PASS  PET            1 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET1  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET1 Finalizing ESMF
20220409 060027.310 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.310 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.310 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.310 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.310 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.310 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.310 INFO             PET2 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.310 INFO             PET2 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.310 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.310 INFO             PET2 ESMF_COMM                   : mpi
20220409 060027.310 INFO             PET2 ESMF_MOAB                   : enabled
20220409 060027.310 INFO             PET2 ESMF_LAPACK                 : enabled
20220409 060027.310 INFO             PET2 ESMF_NETCDF                 : enabled
20220409 060027.310 INFO             PET2 ESMF_PNETCDF                : disabled
20220409 060027.310 INFO             PET2 ESMF_PIO                    : enabled
20220409 060027.310 INFO             PET2 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET2  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET2  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET2  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET2  PASS  PET            2 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET2  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET2 Finalizing ESMF
20220409 060027.314 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.314 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.314 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.314 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.314 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.314 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.314 INFO             PET3 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.314 INFO             PET3 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.314 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.314 INFO             PET3 ESMF_COMM                   : mpi
20220409 060027.314 INFO             PET3 ESMF_MOAB                   : enabled
20220409 060027.314 INFO             PET3 ESMF_LAPACK                 : enabled
20220409 060027.314 INFO             PET3 ESMF_NETCDF                 : enabled
20220409 060027.314 INFO             PET3 ESMF_PNETCDF                : disabled
20220409 060027.314 INFO             PET3 ESMF_PIO                    : enabled
20220409 060027.314 INFO             PET3 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET3  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET3  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET3  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET3  PASS  PET            3 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET3  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET3 Finalizing ESMF
20220409 060027.310 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.310 INFO             PET4 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.310 INFO             PET4 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.310 INFO             PET4 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.310 INFO             PET4 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.310 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.310 INFO             PET4 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.310 INFO             PET4 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.310 INFO             PET4 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.310 INFO             PET4 ESMF_COMM                   : mpi
20220409 060027.311 INFO             PET4 ESMF_MOAB                   : enabled
20220409 060027.311 INFO             PET4 ESMF_LAPACK                 : enabled
20220409 060027.311 INFO             PET4 ESMF_NETCDF                 : enabled
20220409 060027.311 INFO             PET4 ESMF_PNETCDF                : disabled
20220409 060027.311 INFO             PET4 ESMF_PIO                    : enabled
20220409 060027.311 INFO             PET4 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET4  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET4  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET4  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET4  PASS  PET            4 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET4  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET4 Finalizing ESMF
20220409 060027.304 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.304 INFO             PET5 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.304 INFO             PET5 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.304 INFO             PET5 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.304 INFO             PET5 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.304 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.304 INFO             PET5 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.304 INFO             PET5 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.304 INFO             PET5 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.304 INFO             PET5 ESMF_COMM                   : mpi
20220409 060027.305 INFO             PET5 ESMF_MOAB                   : enabled
20220409 060027.305 INFO             PET5 ESMF_LAPACK                 : enabled
20220409 060027.305 INFO             PET5 ESMF_NETCDF                 : enabled
20220409 060027.305 INFO             PET5 ESMF_PNETCDF                : disabled
20220409 060027.305 INFO             PET5 ESMF_PIO                    : enabled
20220409 060027.305 INFO             PET5 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET5  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET5  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET5  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET5  PASS  PET            5 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET5  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET5 Finalizing ESMF
20220409 060027.318 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.318 INFO             PET6 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.318 INFO             PET6 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.318 INFO             PET6 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.318 INFO             PET6 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.318 INFO             PET6 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.318 INFO             PET6 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.318 INFO             PET6 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.318 INFO             PET6 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.318 INFO             PET6 ESMF_COMM                   : mpi
20220409 060027.319 INFO             PET6 ESMF_MOAB                   : enabled
20220409 060027.319 INFO             PET6 ESMF_LAPACK                 : enabled
20220409 060027.319 INFO             PET6 ESMF_NETCDF                 : enabled
20220409 060027.319 INFO             PET6 ESMF_PNETCDF                : disabled
20220409 060027.319 INFO             PET6 ESMF_PIO                    : enabled
20220409 060027.319 INFO             PET6 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET6  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET6  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET6  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET6  PASS  PET            6 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET6  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET6 Finalizing ESMF
20220409 060027.327 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.327 INFO             PET7 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220409 060027.327 INFO             PET7 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220409 060027.327 INFO             PET7 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220409 060027.327 INFO             PET7 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220409 060027.327 INFO             PET7 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220409 060027.327 INFO             PET7 Running with ESMF Version   : v8.3.0b12-1-g7c85d54760
20220409 060027.327 INFO             PET7 ESMF library build date/time: "Apr  9 2022" "00:40:20"
20220409 060027.327 INFO             PET7 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/gfortran_8.3.0_mpi_O_develop
20220409 060027.327 INFO             PET7 ESMF_COMM                   : mpi
20220409 060027.327 INFO             PET7 ESMF_MOAB                   : enabled
20220409 060027.327 INFO             PET7 ESMF_LAPACK                 : enabled
20220409 060027.327 INFO             PET7 ESMF_NETCDF                 : enabled
20220409 060027.327 INFO             PET7 ESMF_PNETCDF                : disabled
20220409 060027.327 INFO             PET7 ESMF_PIO                    : enabled
20220409 060027.327 INFO             PET7 ESMF_YAMLCPP                : enabled
20220409 060027.327 INFO             PET7  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
20220409 060027.327 INFO             PET7  NUMBER_OF_PROCESSORS           8
20220409 060027.388 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
20220409 060027.388 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
20220409 060027.388 INFO             PET7  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
20220409 060027.390 INFO             PET7  PASS  PET            7 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
20220409 060027.392 INFO             PET7  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
20220409 060027.392 INFO             PET7 Finalizing ESMF
