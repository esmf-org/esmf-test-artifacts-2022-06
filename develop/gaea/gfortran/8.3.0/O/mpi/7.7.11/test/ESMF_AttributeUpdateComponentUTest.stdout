build time -- 2022-04-19 00:28:09
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  Beginning Test, file ESMF_AttributeUpdateComponentUTest.F90, line 442
  NUMBER_OF_PROCESSORS           8
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            6 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            7 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            2 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            3 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value1, ESMF_AttributeUpdateComponentUTest.F90, line 526
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value2, ESMF_AttributeUpdateComponentUTest.F90, line 533
  PASS  PET            4 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            5 : Getting an updated Attribute value from a Component test: value = value3, ESMF_AttributeUpdateComponentUTest.F90, line 540
  PASS  PET            0 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            1 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            2 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            3 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            4 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            5 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            6 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  PASS  PET            7 : Getting a lone Attribute from a Component test: value = Lone Attribute, ESMF_AttributeUpdateComponentUTest.F90, line 547
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  Ending Test, file ESMF_AttributeUpdateComponentUTest.F90, line 572
  PET           2  Test Elapsed Time    74.080999999999989       msec.
  PET           3  Test Elapsed Time    72.325999999999993       msec.
  PET           4  Test Elapsed Time    73.906999999999996       msec.
  PET           5  Test Elapsed Time    73.910999999999987       msec.
  PET           6  Test Elapsed Time    72.885999999999996       msec.
  PET           0  Test Elapsed Time    42.588999999999999       msec.
  PET           1  Test Elapsed Time    72.250999999999991       msec.
  PET           7  Test Elapsed Time    73.748999999999981       msec.
